src/common/core.yaml

```diff
@@ -16,13 +16,17 @@ repos:
  - fedora-36-updates
  - fedora
  - fedora-updates
+ #- terra # f37+ only
```

Comment:
This should be uncommented as next is f38

Reply:
done:)